googlefonts-qa/scripts/move-check.sh

```diff
@@ -5,7 +5,7 @@
 # USAGE: 
 # Install requirements with `pip install -U -r googlefonts-qa/requirements.txt`
 # 
-# call this script from the root of your fira code repo, with the absolute path your your local google/fonts repo
+# call this script from the root of your fira code repo, with the absolute path to your local google/fonts repo
```

Comment:
This isn't technically a spelling error, but even old fashioned spell checking tools will flag duplicated words. This is only the third repository I'm piloting this feature against. As such, its handling is still quite alpha. The correction is entirely my choice.